app/src/app/components/specmath-modal/confirm-operation/confirm-operation.component.html

```diff
@@ -0,0 +1,41 @@
+<!-- Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. -->
+
+<div class="modal-step-container">
+  <div class="modal-step-content">
+    <div class="modal-step-instructions">
+      <p class="mat-body-2">Confirm the files used to perform this operation</p>
+    </div>
+    <div *ngIf="inputFilesValid" class="confirm-operation-container">
+      <mat-chip-list id="confirm-operation-chip-container">
+        <div
+          class="modal-step-chip-list-container"
+          *ngFor="let file of operationFiles; let i = index"
+        >
+          <mat-chip>
+            <mat-icon
+              class="modal-step-chip-icon {{
+                defaultsFile && i === 0 ? 'settings' : 'description'
```

Comment:
Consider adding an explicit section for the defaultsFile that is hidden in if the file doesn't exist. If there is too much code duplication, we could always extract a small component for listing the file.

Reply:
Updated, and removed the unused function to collect all files